multimodal-search-tirg/TirgMultimodalEncoder/__init__.py

```diff
@@ -56,8 +56,7 @@ def _get_features(self, data):
             visual_data = np.moveaxis(visual_data, self.channel_axis, self._default_channel_axis)
         textual_data = data[(self.positional_modality.index('text'))]
 
-        visual_data = torch.from_numpy(np.stack(visual_data)).float()
-        textual_data = np.stack(textual_data).tolist()
+        visual_data = torch.stack(visual_data).float()
```

Comment:
In one of my earlier push I had to change below make flow work along with *data in argument and `data` in encode instead of `*data`
From
```
visual_data = torch.stack(visual_data).float()
```

to
```
visual_data = torch.from_numpy(np.stack(visual_data)).float()
textual_data = np.stack(textual_data).tolist()
```